components/Comments/CommentStateChanger.vue

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
CommentStateChanger - for name it seems like it only should do visual thingy, but instead it does have business logic "comment" - could we separate it into component that do toggling stuffs?